src/core/annotation.js

```diff
@@ -439,13 +439,22 @@ class Annotation {
     );
   }
 
-  isHidden(annotationStorage) {
+  isPrintable(annotationStorage) {
```

Comment:
It's a bit confusing now that we have both a public and a private method with this name. If there is no better name, can we maybe add some explanation on how they are different?